code/modules/mob/living/carbon/human/human_say.dm

```diff
@@ -20,12 +23,7 @@
 				return real_name
 		else
 			return real_name
-	if(istype(wear_mask, /obj/item/clothing/mask/infiltrator))
```

Comment:
This is the bit of code you'll need to have readded in order to have the infiltrator balaclava work again. This is not an alternative to ```TRAIT_UNKNOWN``` either. They do different things.